packages/@lightningjs/ui-components/src/components/FocusManager/FocusManager.mdx

```diff
@@ -119,35 +119,52 @@ class FancyFocus extends FocusManager {
 
 ### Properties
```

Comment:
@erautenberg In FocusManager.js, we have getters and setters for props like `selectedIndex`, `items`, `itemPosX`, `itemPosY`. They're not part of the properties array but are specific to FocusManager and can be set/changed and I believe patched to the component thus included in the TS file. That said, should they be included in this properties table?